joschahenningsen/TUM-Live
# Record from browser

### Motivation and Context
Addressing the need for a more seamless recording and uploading process for online educators, this feature allows lecturers to record directly from the browser and instantly upload their lectures to the course.

### Description
This feature is eliminating the need for additional recording software. It allows educators to record both screen and camera feeds simultaneously. The recorded lecture is then being uploaded to the designated course by one click.

### Steps for Testing
Prerequisites:
- 1 Lecturer

1. Log in
2. Navigate to a Course
3. Create new recording lecture
4. Check if recordings are accessable over the lecture page

### Screenshots

https://github.com/joschahenningsen/TUM-Live/assets/17506411/5b4ae298-0502-4b15-aaf7-5d8a3d7a2a5b
